torchaudio/io/_playback.py

```diff
@@ -0,0 +1,55 @@
+import torch
+import torchaudio
+from torchaudio.io import StreamWriter
+from typing import Optional
+
+
+def play_audio(
+    waveform: torch.Tensor,
+    sample_rate: Optional[float],
+    device: Optional[str] = "audiotoolbox",
+    channels_first: Optional[bool] = True,
+    ) -> None:
+    """Play audio through specified or available output device
+
+    Args:
+        src: Tensor of the audio to play. Expected shape: (channels, time)
+            if channels_first is True, (time, channels) otherwise
+        sample_rate: Sample rate of the input audio
+        device: Output device. If None, use available device
+        channels_first: Whether the first dimension corresponds to
+            number of channels or not
+    """
+    dtype = waveform.dtype
+    try:
+        format = {
+            torch.uint8: "u8",
+            torch.int16: "s16",
+            torch.int32: "s32",
+            torch.int64: "s64",
+            torch.float32: "flt",
+            torch.float64: "dbl"
+        }[dtype]
+    except KeyError:
+        raise ValueError(f"Unsupported dtype: {dtype}")
+
+    if device is None:
+        device = torchaudio.utils.ffmpeg_utils.get_output_devices()
+        # need to get device key from dictionary
+    elif device == "audiotoolbox":
+        path = "-"
+
+    if channels_first:
+        waveform = waveform.transpose(0, 1)
+
+    num_frames, num_channels = waveform.size()
```

Comment:
Before doing this we need to check the `waveform.ndim == 2` and raise an error if not.